maint/apply-code-format.sh

```diff
@@ -84,7 +82,6 @@ for ROOT_SUBDIR in ${SRC_DIRS}; do
         # We also skip any dir named "cpu_codegen" in case there are
         # nGraph-generated files lying around from a test run.
         find "${ROOT_SUBDIR}"                                       \
-          -name cpu_codegen -prune -o                               \
```

Comment:
The comment still remains re: cpu_codegen

Reply:
Removed, thanks.